test/integration_test/lib/util/test_author.py

```diff
@@ -0,0 +1,11 @@
+import json
+from lib.util.author import *
+import os
```

Comment:
@tejalwakchoure You don't need to use `import os` here.
Just assign the path of the file to a variable and use it.
Eg.  author_map = './test/integration_test/data/author_uid_map.json' 
Then, just open the file and load it to json.
with open(author_map, 'r') as json_file:
       map_data = json.loads(json_file).keys()

Then you can do the same for your required output file and assert the key values.
Try to run the tests on your system first.
`export PYTHONPATH=.`
`pytest -v`
This will show you the errors in there are any.

Reply:
@hrishikesh-dahiya I tried this, but when author.py prints the name of the output file ("UID map written to ---") , it prints the complete path.

Reply:
But does it matter in testing the script. I mean pytest won't show you the output of the scripts, it only checks and shows the assertions.